### ISSUE TITLE
Negative duration format

### ISSUE THREAD

**Issue by tek-kgong** [dlevs/time#22]
Currently, when we have a negative duration and call  `duration.toString` , it returns a string that is not compatible with certain frameworks like .NET.

For example: 
 ```
const offset = duration.negate({ minutes: 300 })
duration.toString(offset)); // result: PT-300M 
```
.NET expects `-PT300M`.

I am wondering if it is possible to have an option to configure the output of `duration.toString`. 

[Here is a link to another bug report that is related that includes documentation from ISO specification](https://www.postgresql.org/message-id/9q0ftb37dv7.fsf%40gmx.us)

**Comment by dlevs** [comment]
Hi, Thanks for raising this! What do you think of this fix?

https://github.com/dlevs/duration-fns/pull/25

**Comment by tek-kgong** [comment]
Looking good to me. Thanks for the quick response! 

When will the fix be available?

**Comment by dlevs** [comment]
No worries!

It's published now as version 3.0.0
https://www.npmjs.com/package/duration-fns
